spec/lib/completions/cancel_manager_spec.rb

```diff
@@ -3,6 +3,8 @@
 describe DiscourseAi::Completions::CancelManager do
   fab!(:model) { Fabricate(:anthropic_model, name: "test-model") }
 
+  before { WebMock.allow_net_connect! }
```

Comment:
I’m not following this as it means that we are allowing outgoing http connections in the test environment. Not sure if I am missing something here.

Reply:
yes, read downwards, it creates a web server see the current flake, there is a bug in webmock somewhere where it defines local incorrectly.